src/Lecture3.hs

```diff
@@ -92,7 +98,13 @@ weekday to the second.
 >>> daysTo Friday Wednesday
 5
 -}
-daysTo = error "TODO"
+
+daysTo :: Weekday -> Weekday -> Int 
+daysTo day1 day2 = go day1 day2 0
+  where
+    go d1 d2 diff 
+      | d1 == d2 = diff
+      | otherwise = go (next d1) d2 (1 + diff)
```

Comment:
That's a correct implementation 👍🏻 

You can use a bit of math to write it shorter and avoid recursive calls entirely 🙂